setup.cfg

```diff
@@ -0,0 +1,47 @@
+[coverage:report]
+precision = 2
+exclude_lines =
+    @abc.abstractmethod
+    @abc.abstractproperty
+    NotImplementedError
+    pragma: no cover
+    __repr__
+    __str__
+
+[coverage:run]
+branch = True
+source =
+    src
+    tests
+omit =
+    tests/utils.py
+
+[isort]
+atomic = true
+default_section = FIRSTPARTY
+include_trailing_comma = true
+indent = '    '
+known_first_party = airflow_declarative
+known_third_party =
+line_length = 79
+lines_after_imports = 2
+multi_line_output = 3
+order_by_type = true
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+
+[tool:pytest]
+python_files = tests/**/*.py tests/*.py
+
+; Show warnings. Similar to `python -Wd`.
+filterwarnings = d
+
+addopts =
+    --verbose
+    --showlocals
+    --cov=src
+    --cov=tests
```

Comment:
Those cov options are redundant since you specified what to include within coverage config.

Reply:
Indeed, I must have missed that